orchestra/workflow/load.py

```diff
@@ -182,12 +182,6 @@ def load_workflow_version(version_data, workflow, force=False):
         _set_step_dependencies(step, step_data, 'creation_depends_on', Step,
                                workflow_version=version)
 
```

Comment:
we used to be super-cautious when (re)loading a workflow to make sure we don't change the workflow's topology in some fundamental way. that makes sense for task dependencies. for submission dependencies, we're not actually changing the topology, and i needed this for an internal workflow update (closing out certain tasks should rely on more than just a single team member)